web/github_hook.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
A web hook is nice for a beta site, but not for production, we don't want the production site to switch to master instead of being on a release tag every time we merge a PR.